src/kube-runtime/src/init.d/initializer.py

```diff
@@ -100,10 +100,10 @@ def init_plugins(jobconfig, commands, plugins_path, runtime_path, taskrole):
             run_script("{}/{}".format(plugin_base_path, plugin_desc["init-script"]), parameters, plugin_scripts)        
 
         if os.path.isfile(plugin_scripts[0]):
-            commands[0].append("/bin/bash {}".format(plugin_scripts[0]))
+            commands[0].append("/bin/bash {} || exit $?".format(plugin_scripts[0]))
```

Comment:
Since we already have `set -o errexit`, do we still need `|| exit $?` here?

Reply:
This is needed because the generated file doesn't have set -o errexit